app/views/jobseekers/job_applications/_messages.html.slim

```diff
@@ -0,0 +1,22 @@
+.govuk-grid-row
+  .govuk-grid-column-full
+    - if @messages.any?
```

Comment:
unsure about putting this logic here (don't allow new messages unless we have any). Think this should be in the model layer somehow, it's pretty obscure IMHO in this location

Reply:
Well this is decide whether to show a form, or to show content which tells the user that no messages exist yet so I think it's necessary in this case.

Reply:
why do we show a form based on whether they are current messages? When we have rejection messages won't that need changing? Hence I wondered if it should be a model concern rather than a view one